plugins/slack_app/app/helpers/slack_app_helper.rb

```diff
@@ -0,0 +1,12 @@
+require 'uri'
+
+module SlackAppHelper
+  def oauth_url(scopes)
+    redir_url = URI.encode url_for(controller: :slack_app, action: :oauth, only_path: false)
```

Comment:
arg yes ... that wont work ... need to use url_for

On Sun, Jul 17, 2016 at 3:56 PM, Ben Straub <notifications@github.com>
wrote:

> In plugins/slack_app/app/helpers/slack_app_helper.rb
> <https://github.com/zendesk/samson/pull/1130#discussion_r71091516>:
>
> > @@ -0,0 +1,12 @@
> > +require 'uri'
> > +
> > +module SlackAppHelper
> > +  def oauth_url(scopes)
> > +    redir_url = URI.encode url_for(controller: :slack_app, action: :oauth, only_path: false)
>
> Nice tip on to_query. Can you help me out with the routing stuff? These
> aren't mounted as resources, so I'm not sure how to get to
> /slack_app/oauth in a more succinct way (fairly new to Rails here).
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/d008e855296f051e83d429a560e4e921b94fd917#r71091516>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZyHZJlBGDS9dpZ1es7ky8obRb0AZks5qWrMCgaJpZM4JIRHk>
> .
>